views/js/controller/creator/creator.js

```diff
@@ -78,14 +77,17 @@ define([
           * @param {Object} options.labels - the list of item's labels to give to the ItemView
           * @param {Object} options.routes - action's urls
           * @param {Object} options.categoriesPresets - predefined category that can be set at the item or section level
-          * @param {Boolean} [options.guidedNavigation  = false]- feature flag for the guided navigation
+          * @param {Boolean} [options.guidedNavigation=false] - feature flag for the guided navigation
           */
-        start : function(options){
-            var self = this;
-            var $container = $('#test-creator');
-            var $saver = $('#saver');
-            var binder, binderOptions, modelOverseer;
+        start(options){
+            const self = this;
+            const $container = $('#test-creator');
+            const $saver = $('#saver');
+
             var creatorContext;
```

Comment:
What do you think about using let instead of var? It can be good as it prevents re declaration, usage before declaration etc